Flow.Launcher.Core/Resource/Internationalization.cs

```diff
@@ -58,6 +58,7 @@ internal void AddPluginLanguageDirectories(IEnumerable<PluginPair> plugins)
 
         private void LoadDefaultLanguage()
         {
+            RemoveOldLanguageFiles();
```

Comment:
Removes language files loaded before any plugins were loaded. Prevents the language Flow started in from overwriting English if the user switches back to English